src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2848,4 +2955,123 @@ public function is_development_environment() {
 		return in_array( wp_get_environment_type(), array( 'development', 'local' ), true );
 	}
 
+	/**
+	 * Determines whether to suggest using a persistent object cache.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @global wpdb $wpdb WordPress database abstraction object.
+	 *
+	 * @return bool Whether to suggest using a persistent object cache.
+	 */
+	public function should_suggest_persistent_object_cache() {
+		global $wpdb;
+
+		if ( is_multisite() ) {
+			return true;
+		}
+
+		/**
+		 * Filters to force suggestion to use a persistent object cache and bypass threshold checks.
+		 *
+		 * @since 6.1.0
+		 *
+		 * @param bool $suggest Whether to suggest using a persistent object cache.
+		 */
+		if ( apply_filters( 'site_status_suggest_persistent_object_cache', false ) ) {
```

Comment:
@manuelRod A bit late, but I am just realizing that this should rather follow a more appropriate short-circuiting behavior: Right now, the only way to short-circuit would be to actually recommend a persistent object cache. But it also needs to be possible to run custom logic that would result in _not_ recommending a persistent object cache. So I think the default value here should be `null`, and any boolean result should cause the method to return early with that result.

This is a trivial change, which is compatible with the current behavior, it just expands it to also allow returning `false` to short-circuit.